web_m2x_options_manager/models/m2x_create_edit_option.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.cache import ormcache
+from odoo.tools.safe_eval import safe_eval
+
+
+class M2xCreateEditOption(models.Model):
+    _name = "m2x.create.edit.option"
+    _description = "Manage Options 'Create/Edit' For Fields"
+
+    ir_model_field_id = fields.Many2one(
```

Comment:
Not blocking.. but here I would simply field names for the sake of readability

- `ir_model_field_id` -> `field_id`
- `ir_model_field_name` -> `field_name`
- `ir_model_id` -> `model_id`
- `ir_model_name` -> `model_name`